src/org/thoughtcrime/securesms/ConversationListActivity.java

```diff
@@ -100,6 +101,22 @@ public void onDestroy() {
     super.onDestroy();
   }
 
+  @SuppressLint("RestrictedApi")
```

Comment:
Just curious as to what this warning is for?

Reply:
The usage of the method setOptionalIconsVisible(true) requires that line to suppress a warning from lint. They set up lint to make it throw a warning when you try to call that method outside of certain classes, but this was the only way to do it.